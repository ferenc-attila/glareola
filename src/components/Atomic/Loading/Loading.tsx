import { ActivityIndicator, StyleSheet } from 'react-native';

import i18n from '../../../localization';
import { COLORS } from '../../../styles';

export const Loading = () => {
    return (
        <ActivityIndicator
            style={styles.loading}
            size='large'
            color={COLORS.BACKGROUND.BUTTON}
            accessibilityLabel={i18n.t('LOADING')}
        />
    );
};

const styles = StyleSheet.create({
    loading: {
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center',
        alignSelf: 'center',
    },
});
