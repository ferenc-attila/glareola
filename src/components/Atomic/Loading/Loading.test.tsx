import renderer from 'react-test-renderer';

import { Loading } from './Loading';

describe('Loading', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Loading />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
