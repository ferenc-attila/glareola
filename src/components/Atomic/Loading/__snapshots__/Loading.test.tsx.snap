// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<ActivityIndicator
  accessibilityLabel="loading"
  color="#4d9460"
  size="large"
  style={
    {
      "alignItems": "center",
      "alignSelf": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
/>
`;
